marklogic-data-hub/src/test/java/com/marklogic/hub/flow/impl/FlowRunnerTest.java

```diff
@@ -82,8 +82,10 @@ public void testRunFlow() {
         XMLDocumentManager docMgr = stagingClient.newXMLDocumentManager();
         DocumentMetadataHandle metadataHandle = new DocumentMetadataHandle();
         docMgr.readMetadata("/ingest-xml.xml", metadataHandle);
-        DocumentMetadataHandle.DocumentPermissions perms = metadataHandle.getPermissions();
-        Assertions.assertEquals(2, perms.get("data-hub-operator").size());
+        DocumentMetadataHandle.DocumentPermissions permissions = metadataHandle.getPermissions();
+        Assertions.assertEquals(2, permissions.get("data-hub-operator").size());
+        Assertions.assertTrue(permissions.get("data-hub-operator").contains(DocumentMetadataHandle.Capability.READ));
```

Comment:
The underlying ml-javaclient-util library doesn't yet support "node_update", as an FYI. So we may not want to list it yet in the tooltip.

Reply:
I removed 'node-update' from the list valid capabilities (ui will throw a validation error if capability is set to 'node-update'), tooltip and error message.

Reply:
I think that's fine for now, we'll likely need to update all of DHF later to support "node update" across the board.